corehq/apps/app_manager/models.py

```diff
@@ -5687,6 +5692,22 @@ def get_subcase_types(self, case_type):
                 if m.case_type == case_type
                 for t in m.get_subcase_types()}
 
+    @memoized
+    def grid_display_for_some_modules(self):
+        return self.grid_menu_toggle_enabled() and self.grid_form_menus == 'some'
+
+    @memoized
+    def grid_display_for_all_modules(self):
+        return self.grid_menu_toggle_enabled() and self.grid_form_menus == 'all'
+
+    def grid_menu_toggle_enabled(self):
```

Comment:
The more standard way to check a toggle is `toggles.GRID_MENUS.enabled(self.domain)`